src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -1023,31 +1023,33 @@ var requirejs = (function() {
 		});
 	}
 
-	async showMarkupPreview(initialization: IMarkupCellInitialization) {
+	async showMarkupPreview(newContent: IMarkupCellInitialization) {
 		if (this._disposed) {
 			return;
 		}
 
-		const entry = this.markupPreviewMapping.get(initialization.cellId);
+		const entry = this.markupPreviewMapping.get(newContent.cellId);
 		if (!entry) {
-			return this.createMarkupPreview(initialization);
+			return this.createMarkupPreview(newContent);
 		}
 
-		const sameContent = initialization.content === entry.content;
-		if (!sameContent || !entry.visible) {
+		const sameContent = newContent.content === entry.content;
+		const sameMetadata = newContent.metadata === entry.metadata;
```

Comment:
You may want to use `object.equals` for this instead:

https://github.com/microsoft/vscode/blob/395380a6b7560f9d67323b98ebdd392805f819b7/src/vs/base/common/objects.ts#L117